packages/plugins/color-picker/admin/src/global.d.ts

```diff
@@ -1,2 +1,3 @@
 declare module '@strapi/helper-plugin';
 declare module '@strapi/design-system';
+declare module '@strapi/icons';
```

Comment:
Thanks for this, I was investigating that error too, but I was still trying to figure out the other error for importing the component:

An import path can only end with a '.tsx' extension when 'allowImportingTsExtensions' is enabled

I tried what was recommended for the tsconfig which suppressed the error but then the linter complained with:

error  Parsing error: Unknown compiler option 'allowImportingTsExtensions'

🤔

Reply:
I think that was a mistake I made setting `node16` as the moduleResolution for all packages on this branch to allow the imports from `@strapi/design-system/v2` – I think this will now be resolved, thank god for a CI ey?

Reply:
oh got it, nice! Thanks for your help on this.